Source/WebKit/WebProcess/WebPage/RemoteLayerTree/GraphicsLayerCARemote.cpp

```diff
@@ -135,5 +136,9 @@ RefPtr<WebCore::GraphicsLayerAsyncContentsDisplayDelegate> GraphicsLayerCARemote
     return adoptRef(new GraphicsLayerCARemoteAsyncContentsDisplayDelegate(*WebProcess::singleton().parentProcessConnection(), m_context->drawingAreaIdentifier(), primaryLayerID()));
 }
 
+void GraphicsLayerCARemote::setRemoteFrame(RemoteFrame& frame)
+{
+    WebProcess::singleton().parentProcessConnection()->send(Messages::RemoteLayerTreeDrawingAreaProxy::LayerWillBeUsedForRemoteFrame(primaryLayerID(), frame.frameID()), m_context->drawingAreaIdentifier());
```

Comment:
I don't think this should IPC. It should just store some ID on the layer which gets to the UI process, where layers are glued together.